src/wmnds/assets/sass/_utilities.scss

```diff
@@ -19,6 +19,79 @@
   &.wmnds-grid--align-center {
     align-items: center;
   }
+
+  &.wmnds-grid--align-stretch {
+    align-items: stretch;
+  }
+
+  // Column spacing
+  &.wmnds-grid--spacing {
+    &-xs {
+      width: auto;
+      margin-right: -$size-xs;
+      margin-left: -$size-xs;
```

Comment:
@houbly Yeah that makes sense, I've had a look at each solution. The first would give us the same problem unless we implemented something similar to bootstrap instead of using `wmnds-grid` throughout the page. 

The second would be easier yes, but would need to be implemented for all (or the most used) `wmnds-col-*` classes for each size so I'm looking into making a sass mixin for it.